src/sdk/model/policy.ts

```diff
@@ -74,14 +85,77 @@ export class PolicyBuilder {
   constructor(private _entityUuid: string, private _entityDomain: EntityDomainType, private _type: PolicyType) {
   }
 
+  /**
+   * Set entity uuid.
+   * @param {string} uuid
+   * @returns {PolicyBuilder}
+   */
+  setEntityUuid(uuid: string): PolicyBuilder {
+    this._entityUuid = uuid;
+    return this;
+  }
+
+  /**
+   * Set entity domain type.
+   * @param {EntityDomainType} entityDomainType
+   * @returns {PolicyBuilder}
+   */
+  setEntityDomainType(entityDomainType: EntityDomainType): PolicyBuilder {
+    this._entityDomain = entityDomainType;
+    return this;
+  }
+
+  /**
+   * Set policy type.
+   * @param {PolicyType} type
+   * @returns {PolicyBuilder}
+   */
+  setPolicyType(type: PolicyType): PolicyBuilder {
```

Comment:
it would be better to not have these new setters because you're opening the door to missing validation issues. For example, imagine that a bunch of permissions are added for the VM domain and then the entity domain type is later changed to VAPP before being submitted.

Reply:
Humm ok. I'll remove them. It was just easier to use (specially for my tests) but you're right, it's not good.